script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -625,22 +755,24 @@ Using the bare minimum amount of force, she repels me with her middle finger alo
 [sha:b800317f3b276359a1908da68453cfa6a5dd5a4d]{
 -- Page 152, Offset 35679.
 -- 背中から鉄柵に衝突する。
-My back collides with the iron fence.
+I slam back-first into the iron fence.
 }
 [sha:0a3c52de4870b4310020ebe5ee8be09f154967c7]{
 -- Page 152, Offset 35680.
 -- 横腹の感覚がない。横隔膜の損傷で呼吸が出来ない。
-My sides have gone numb. I can't breathe because of the damage to my diaphragm.
+I've lost all feeling where she hit me.
+ My diaphragm is so damaged I can't even breathe.
 }
 [sha:6fc76620e7c9f390f4b7c40d7e4afc4af6204021]{
 -- Page 152, Offset 35681.
 -- 加えてこの音。
-Not to mention this noise.
+And then there's the noise―――
 }
 [sha:c007db18213045892ac023a548dfdb8e7cfb0cbd]{
 -- Page 152, Offset 35682.
 -- 鉄柵に触れた途端、脳をかき回す音に襲われる。
-The moment I crash into the iron fence, the assault of noise makes my brain start churning.
+The moment I came into contact with the iron fence,
+ I was bombarded by a noise that threatens to turn my brain to mush.
 }
 [sha:18c797b30d16b26ee3eebd218f830ca8b52d9e1e]{
 -- Page 153, Offset 35683.
```

Comment:
I feel like "I see" is very unnecessary. It's more about the gravity of the situation.
>When I open my eyes, the Dead have already started reaching toward me.